incident_viewer_demo/src/IspSelect/react-select.scss

```diff
@@ -0,0 +1,371 @@
+/**
+ * React Select
```

Comment:
Perhaps add a docstring that says what this is used for.

Reply:
This component comes from the existing MLab client codebase.